datahub-web-react/src/app/home/HomePageHeader.tsx

```diff
@@ -166,38 +146,32 @@ export const HomePageHeader = () => {
         }
     };
 
-    // fetch some results from each entity to display search suggestions
-    const allSearchResultsByType = useGetAllEntitySearchResults({
-        query: '*',
-        start: 0,
-        count: 20,
-        filters: [],
+    // Fetch results
+    const { data: searchResultsData, loading: searchResultsLoading } = useGetSearchResultsForMultipleQuery({
+        variables: {
+            input: {
+                types: [],
+                query: '*',
+                start: 0,
+                count: 20,
+                filters: [],
+            },
+        },
     });
 
-    const suggestionsLoading = Object.keys(allSearchResultsByType).some((type) => {
-        return allSearchResultsByType[type].loading;
-    });
+    const searchResultsToShow = useMemo(() => {
+        let result: string[] | undefined = [];
+        if (!searchResultsLoading) {
```

Comment:
We can also just say...

if (searchResultsData) {

....

}

Reply:
This will also simplify the dependencies required for the useMemo

Reply:
Fixed and Commited